include/swift/SILOptimizer/Analysis/ARCAnalysis.h

```diff
@@ -186,8 +186,12 @@ class ConsumedArgToEpilogueReleaseMatcher {
   RCIdentityFunctionInfo *RCFI;
   ExitKind Kind;
   llvm::SmallMapVector<SILArgument *, ReleaseList, 8> ArgInstMap;
+  /// TODO: Replace this with ThisBlock. 
   bool HasBlock = false;
 
+  /// Eventually this will be used in place of HasBlock.
+  SILBasicBlock *ThisBlock;
```

Comment:
This comment is out of date. I will remove the HasBlock. actually i will be doing a refactoring on the retain release matcher sometime next week.